docs/guides/nrfconnect_examples_configuration.md

```diff
@@ -0,0 +1,136 @@
+# Configuring nRF Connect SDK examples
+
+When you tried to build nRF Connect SDK example applications like
+[lock-app](../../examples/lock-app/nrfconnect/README.md),
+[lighting-app](../../examples/lighting-app/nrfconnect/README.md) or
+[pigweed-app](../../examples/pigweed-app/nrfconnect/README.md), you could notice
+that some default configuration is used then. Probably after testing an example
```

Comment:
> Probably after testing an example solution you may want to change something

I think i would remove this sentence.

Reply:
I didn't remove it, but I reformatted it to be more informative.